src/components/v2/values/chartValuesDiff/ChartValuesView.tsx

```diff
@@ -0,0 +1,1128 @@
+                                    chartRepoName: '',
+                                    chartId: 0,
+                                    chartName: _releaseInfo.deployedAppDetail.chartName,
+                                    version: _releaseInfo.deployedAppDetail.chartVersion,
+                                    deprecated: false,
+                                },
+                                selectedVersionUpdatePage: _chartVersionData,
+                                chartVersionsData: [_chartVersionData],
+                                chartValues: _chartValues,
+                                modifiedValuesYaml: YAML.stringify(JSON.parse(_releaseInfo.mergedValues)),
+                            },
+                        })
+                    }
+                })
+                .catch((errors: ServerErrors) => {
+                    showError(errors)
+                    dispatch({
+                        type: 'multipleOptions',
+                        payload: {
+                            isLoading: false,
+                            errorResponseCode: errors.code,
+                        },
+                    })
+                })
+        }
+
+        if (!isDeployChartView) {
+            getDeploymentHistory(appId, isExternalApp)
+                .then((deploymentHistoryResponse: ChartDeploymentHistoryResponse) => {
+                    const _deploymentHistoryArr =
+                        deploymentHistoryResponse.result?.deploymentHistory?.sort(
+                            (a, b) => b.deployedAt.seconds - a.deployedAt.seconds,
+                        ) || []
+                    dispatch({
+                        type: 'deploymentHistoryArr',
+                        payload: _deploymentHistoryArr,
+                    })
+                })
+                .catch((e) => {})
+        }
+    }, [])
+
+    useEffect(() => {
+        if (
+            commonState.chartValues &&
+            ((commonState.chartValues.id && commonState.chartValues.chartVersion) ||
+                (isExternalApp && commonState.releaseInfo))
+        ) {
+            dispatch({ type: 'fetchingValuesYaml', payload: true })
+            if (commonState.chartValues.id && commonState.chartValues.chartVersion) {
+                getChartValues(commonState.chartValues.id, commonState.chartValues.kind)
+                    .then((response) => {
+                        dispatch({
+                            type: 'multipleOptions',
+                            payload: {
+                                fetchingValuesYaml: false,
+                                modifiedValuesYaml: response.result.values || '',
+                            },
+                        })
+
+                        if (
+                            commonState.activeTab === 'manifest' &&
+                            (!isExternalApp || (isExternalApp && commonState.installedAppInfo)) &&
```

Comment:
can we write (!isExternalApp || (isExternalApp && commonState.installedAppInfo)) to (!isExternalApp || commonState.installedAppInfo)?